exercises/5-operations-update-vault/ReadMe.md

```diff
@@ -1,12 +1,15 @@
 Vault Update
 ===
-In this exercise, we will update the vault version of the primary cluster to the new version of Vault.
+In this exercise, we will update the vault version to the new version of Vault.
 We will use ArgoCD value overrides. Normally we would update the version in the helm chart values in the git repository, but for this exercise we will use overrides, since we don't have access to the git repository.
 
+Read the [upgrade guide](https://developer.hashicorp.com/vault/docs/upgrading#enterprise-replication-installations) first.
+
 ## Steps
-1. Update the vault version in the ArgoCD value overrides
-2. Delete the vault pod in the primary cluster
-3. Verify that the vault pod is running the new version
-4. Verify that the vault pod in the secondary cluster is still running the old version
+1. Update the vault version in the ArgoCD value overrides for the DR cluster
```

Comment:
I fixed the order of applying the upgrade steps, so that the secondary cluster gets upgraded before the primary cluster.. This is also what is suggested in https://developer.hashicorp.com/vault/docs/upgrading#enterprise-replication-installations